kurl_proxy/Makefile

```diff
@@ -1,5 +1,11 @@
 SHELL := /bin/bash -o pipefail
-export GO111MODULE=on
+export GO111MODULE=off
```

Comment:
I don't understand this change, is this something from your env?

Reply:
i was experimenting with something. makes the build a lot faster. i backed it out